lib/matplotlib/figure.py

```diff
@@ -1220,12 +1220,16 @@ def colorbar(
                 fig.colorbar(cm.ScalarMappable(norm=norm, cmap=cmap), ax=ax)
 
         cax : `~matplotlib.axes.Axes`, optional
-            Axes into which the colorbar will be drawn.
+            Axes into which the colorbar will be drawn.  If `None`, then space
+            will be stolen from the Axes(s) specified in *ax*
```

Comment:
```suggestion
            Axes into which the colorbar will be drawn.  If `None`, then a new
            Axes is created and the space for it will be stolen from the Axes(s)
            specified in *ax*.
```